tests/ctst/features/quotas/Quotas.feature

```diff
@@ -96,3 +96,43 @@ Feature: Quota Management for APIs
             | 100        | 200         | 0            | IAM_USER |
             | 100        | 0           | 200          | IAM_USER |
             | 100        | 200         | 200          | IAM_USER |
+
+    @2.6.0
+    @PreMerge
+    @Quotas
+    @CronJob
+    @DataDeletion
+    @NonVersioned
+    Scenario Outline: Quotas are affected by deletion operations between count items runs
+        Given an action "DeleteObject"
+        And a permission to perform the "PutObject" action
+        And a STORAGE_MANAGER type
+        And a bucket quota set to 10000 B
+        And an account quota set to 10000 B
+        And an upload size of 1000 B for the object "obj-1"
+        And a bucket quota set to <bucketQuota> B
+        And an account quota set to <accountQuota> B
+        And a <userType> type
+        And an environment setup for the API
+        And an "existing" IAM Policy that "applies" with "ALLOW" effect for the current API
+        When I wait 3 seconds
+        And I PUT an object with size <uploadSize>
+        Then the API should "fail" with "QuotaExceeded"
+        When the "count-items" cronjobs completes without error
```

Comment:
No real other way to test this scenario without this step today (which will need removal with the future scuba implementation / drop of count items)

Reply:
countItems is used to reset in-flights, right?

should we not simply create another test for the followup? it would achieve the same purpose, right?

Reply:
I am not sure to understand what you mean by "another test"? We are already testing all cases we can in Zenko, this one is mostly to ensure that after resetting the inflights, so we have negative inflight (but positive current + inflight), we cannot bypass the quota

Reply:
what i mean is in the same scenario, after "setup",
* we try to write and expect to be rejected by "inflights" → first scenario
* then we run count-items, wait and delete object (generate negative inflight) → second scenario

Reply:
If I understand correctly, you want to handle the same bucket and objects in a separate scenario, so we also run count items in-between, and have 2 distinct scenario, one focusing on inflights, and one on the actual negative inflights?

If yes then this is against the best practices of cucumberjs test: they should all work with their set of resources, and not rely on one another.

If we just want a scenario specific to the "negative inflight", then we can just delete these lines:
```
        When I wait 3 seconds
        And I PUT an object with size <uploadSize>
        Then the API should "fail" with "QuotaExceeded"
```
However having them also ensure we properly clean the inflights when the request fails, and thus it should not impact what we have after count items. This is the scenario close to what a human would do: put object, hit the quota, then the count items runs, and finally, we should be able to delete data and be allowed to write again.